components/appstudio/application-service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: Component
+metadata:
+  name: application-service-component
+  annotations:
+    pipelinesascode: "1"
+    skipprcreation: "1"
```

Comment:
This annotation is a proposal to have the build-service to skip the creation of the PR that sets up the basic PaC resources in the related repository.

On the tests made, however, no PRs were created since the Github App did not have access to the redhat-appstudio org repositories. No errors were found in the build-service logs or the Component's status.

This might indicate that this proposed change to the build-service is not necessary.